packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -323,6 +323,8 @@ export interface IGarbageCollectionRuntime {
 
 /** Defines the contract for the garbage collector. */
 export interface IGarbageCollector {
+	/** Tells the time at which session expiry started */
```

Comment:
Nit: I'd give more context here, that this is set only when continuing a session that was interrupted for an unknown amount of time or however you would say it.